modules/EnsEMBL/Web/Configuration/Gene.pm

```diff
@@ -0,0 +1,70 @@
+
+=head1 LICENSE
+
+Copyright [2009-2022] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package EnsEMBL::Web::Configuration::Gene;
+
+use List::MoreUtils qw(any);
+
+sub modify_tree {
+  my $self         = shift;
+  my $hub          = $self->hub;
+  my $species_defs = $hub->species_defs;
+  my $species = $hub->species;
+
+  my $species_production_name = $species_defs->SPECIES_PRODUCTION_NAME;
+
+  $Data::Dumper::Maxdepth = 6;
```

Comment:
This might cause compilation errors.

Reply:
Dang it! 😠

Reply:
Deleted and pushed (https://github.com/EnsemblGenomes/eg-web-metazoa/commit/6b5c499751b1125c348f892d157e9df8301ba5e7)